exercises/printFavorites.js

```diff
@@ -0,0 +1,20 @@
+let readlineSync = require('readline-sync')
+
+function printFavorites() {
```

Comment:
Make `printFavorites` a function that takes in a list of favorites and prints them. This function shouldn't care what those favorites are, which user they belong to, or whether they found their way into the computer via interactive keyboard input, command line arguments, the contents of a file on disk, a network connection, or any thing else.